data/autoyast_opensuse/autoyast_wicked_x86_64.xml.ep

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0"?>
+<!DOCTYPE profile>
+<profile xmlns="http://www.suse.com/1.0/yast2ns" xmlns:config="http://www.suse.com/1.0/configns">
+  <add-on>
+    <add_on_products config:type="list">
+      % for my $repo (split(/,/, $get_var->('AUTOYAST_ADD_REPOS',''))) {
```

Comment:
@rwx788  WDYT ? is it right way to do it ?

Reply:
TBH, we have other variables to control add-ons, actually too many of them, like `SCC_ADDONS`, just `ADDONS`. And for SLE 15 using urls will work only with full medium. Other than that, it's fine.

Reply:
Yes thanks for the hint. That was my expression as well. And none of them really fits my need.

There is actually also that `MAINT_TEST_REPO` variable which almost doing the same. But I didn't like the approach that we add none common variables into that template replacement logic if it isn't needed.

I also see quite often REPO_1, REPO_2 ... variables. Maybe we could use these too... 

For now I would just keep my logic. It doesn't influence any other and for me, this is what the template logic is for.